config.md

```diff
@@ -49,6 +49,14 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 }
 ```
 
+### Example (Windows)
+
+```json
+"root": {
+    "path": "\\\\?\\Volume{ec84d99e-3f02-11e7-ac6c-00155d7682cf}"
```

Comment:
>  Trailing slash on those is almost always optional, and IMO there's no additional value in being explicit about that here.

I'd agree with that if the docs for volume GUID paths didn't [require a trailing backslash in most cases][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/849#discussion_r118129096